crates/revmc-worker/src/tests/worker.rs

```diff
@@ -46,15 +41,15 @@ fn fib_call_data() -> Bytes {
 
 #[test]
 fn test_compiler_cache_retrieval() {
-    let ext_worker = EXTCompileWorker::new(1, 3, 128);
+    let ext_worker = Arc::new(EXTCompileWorker::new(1, 3, 128));
```

Comment:
can we move wrapping with Arc in EXTCompilerWorker::new? instead of doing outside. we should force smart pointer to ensure thread-safe

Reply:
Okay, but I'm not sure if it's a good practice

Reply:
done

Reply:
yeah, so it is good to force wrapping arc pointer in creating instance